### PR TITLE
 Implemented #1715 / Show wallet locked warning in host command

### PR DESCRIPTION
**Pull request by chrsch**

### CODE REVIEW

File: cmd/siac/hostcmd.go [hunk 1]
```diff
@@ -283,6 +283,17 @@ RPC Stats:
 			currencyUnits(totalRevenue))
 	}
 
+	// if wallet is locked print warning
+	walletstatus := new(api.WalletGET)
+	walleterr := getAPI("/wallet", walletstatus)
+	if err != nil {
+		die("Could not get wallet status. A working wallet is needed in order to operate your host. Error:", walleterr)
```
Comment (lukechampine) [inline cmd/siac/hostcmd.go]: I'm not sure if `die` is appropriate here; a printed warning (like below) is probably sufficient

File: cmd/siac/hostcmd.go [hunk 2]
```diff
@@ -283,6 +283,17 @@ RPC Stats:
...
+	} else {
+		if !walletstatus.Unlocked {
+			fmt.Println("\nWARNING:\n	In order to operate your host you need to unlock your wallet using the command \"siac wallet unlock\"")
+		}
+	}
```
Comment (lukechampine) [inline cmd/siac/hostcmd.go]: you can save some lines here by putting `else if` on the same line. Also, I don't think it's necessary to specify the exact command. How about `"\nWarning: Your wallet is locked. You must unlock your wallet for the host to function properly."`

**Comment by lukechampine** [comment]
Thanks! We can merge this after a bit of tweaking.

**Comment by chrsch** [comment]
Thanks for review & suggestions!

**Pull request merged**